test/cql-pytest/test_tablets.py

```diff
@@ -127,3 +127,50 @@ def verify_tablets_presence(expected:bool=True):
 
     cql.execute(f"DROP TABLE {test_keyspace}.{table_name}")
     verify_tablets_presence(expected=False)
+
+
+def _test_tablets_are_dropped_when_dropping_table_with_view(cql, keyspace_name, attempt_drop_table:bool=False):
+    table_name = unique_name()
+    schema = "pk int PRIMARY KEY, c int"
+    # new_test_table is not used since we want to test a failure to drop the table
+    cql.execute(f"CREATE TABLE {keyspace_name}.{table_name} ({schema})")
+    view_name = unique_name()
+    where = "c is not null and pk is not null"
+    view_pk = "c, pk"
+    cql.execute(f"CREATE MATERIALIZED VIEW {keyspace_name}.{view_name} AS SELECT * FROM {table_name} WHERE {where} PRIMARY KEY ({view_pk})")
+
+    def verify_tablets_presence(table_expected:bool=True, view_expected:bool=True):
```

Comment:
It seems like you repeat the same function verify_tablets_presence() multiple times. Maybe it makes sense to make it into a outer-scope function (in test_tablets.py, not in util.py!)  instead of an inner function? It can get the table name as a parameter (and in this code, it will be called twice, once for the table and once for the view).

Reply:
ok, I'll take a stab at it